requirements.txt

```diff
@@ -1,5 +1,5 @@
 cookiecutter==0.9.0
-Django >= 1.4, < 1.5
+Django==1.8.4
```

Comment:
@nedbat Some requirement in edx-platform is pulling in the xblock-sdk and `install_requires` in `setup.py` here is set to the list in `requirements.txt` (see below). This will result in warnings of the form:
```
2015-10-27 07:39:35,344 WARNING 9681 [xblock.plugin] plugin.py:147 - Unable to load XBlock 'debugchild'
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/src/xblock/xblock/plugin.py", line 144, in load_classes
    yield (class_.name, cls._load_class_entry_point(class_))
  File "/edx/app/edxapp/venvs/edxapp/src/xblock/xblock/plugin.py", line 73, in _load_class_entry_point
    class_ = entry_point.load()
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2354, in load
    self.require(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2371, in require
    items = working_set.resolve(reqs, env, installer)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 844, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
VersionConflict: (Django 1.8.4 (/edx/app/edxapp/venvs/edxapp/lib/python2.7/site-packages), Requirement.parse('Django<1.5,>=1.4'))
```